weblate/trans/models/announcement.py

```diff
@@ -77,6 +78,13 @@ class Announcement(models.Model):
         blank=True,
         on_delete=models.deletion.CASCADE,
     )
+    project_category = models.ForeignKey(
```

Comment:
Why not just `category`?

Reply:
Hi @nijel, I named it `project_category` to avoid confusion with `category` that is already existing in this model.  The existing `category` stores the color scheme (info, warning, danger, success) of the announcement.

Reply:
Good point, I didn't notice it. But for consistency I'd prefer to use category for the same thing. So it's probably time to rename existing category here to something else. Maybe call that severity? That should probably be done in a separate pull request

Reply:
Yes @nijel, we can rename it to a more relevant names. We can name the existing category field as `severity ` or `type`.

Reply:
I think severity is a good choice here

Reply:
Hi @nijel, the `category` field in the `Announcement` model is now renamed to `severity`. The `category` field is now a foreign key to the `Category` model.